### ISSUE TITLE
MagicKeys preference pane says that update is available

### ISSUE THREAD

**Issue by alexreg** [treasurebox/MagicKeys#13]
I'm using v1.4.1 according to the preference pane, and it says a new version is available, and directs me to the website, but the version I just downloaded from the website (again) is also v1.4.1, and after installing it, the preference pane still tells me that an update is available.

**Comment by treasurebox** [comment]
Dear Alexander,




  Thank You for your letter! Really, the newest version is 1.4.2. I am not sure why it does not update, most likely it is a bug in System Preferences.app itself. I suggest you temporarily remove via right click on it’s icon, select remove in the context menu, then reinstall.

  Also, are you trying to download it from the ‘Download’ link at http://www.gearmusicplayer.com/extras.html  ?




All the best,

Zsolt

**Comment by alexreg** [comment]
Hi Zsolt,

Thanks again for your quick reply. In fact, it did update correctly to v1.4.2, but it just didn’t show right away. Weird. Maybe I had to restart System Preferences to notice? But I think it was already closed when I installed. Anyway, never mind. :-)

Cheers,
Alex

**Comment by alexreg** [comment]
Hi again @zsszatmari. I still use Gear and MagicKeys here (on macOS Mojave now), and I noticed that it thinks an update for 1.4.2 is available, but evidently there is no newer version... this is a mistake, right?

Thanks,  Alex

**Comment by zsszatmari** [comment]
Hi! Well, I admit that it is a mess now as I didn’t have time for this project. I’ve uploaded 1.4.2 to https://github.com/zsszatmari/MagicKeys/releases/ just now, but I am not sure that it works on Mojave. If it did not before, than certainly this release won’t be of help.

**Comment by alexreg** [comment]
@zsszatmari Ah okay. Do you still maintain Gear though? Is this necessary for Gear on Mojave?

**Comment by alexreg** [comment]
Incidentally, the preference pane still thinks an update is available, and the "get it" link directs to a defunct website...

**Comment by zsszatmari** [comment]
Yes, Gear is maintained. Media keys will not work without Magic Keys on Mojave, then again, it may not work with it either. I will try to fix the situation when I get there, currently I am a bit overwhelmed.

**Comment by alexreg** [comment]
@zsszatmari No worries at all. Thanks for your efforts on this great software!
